internal/server/data/access_key.go

```diff
@@ -17,6 +16,24 @@ import (
 	"github.com/infrahq/infra/uid"
 )
 
+type accessKeyTable models.AccessKey
+
+func (a accessKeyTable) Table() string {
+	return "access_keys"
+}
+
+func (a accessKeyTable) Columns() []string {
```

Comment:
They are generated for now, but I don't think we want to force everyone to use the generated code to make small changes. It often ends up being more work to figure out the code generation than to simply add the new field to these 3 methods. It should be fine for someone to manually edit these when we add one or two fields, and use code gen when we make larger changes. It would be nice to support both workflows.

I think this is one of the last remaining open questions about this approach.

> If we create them maybe we should have a shared list of common columns for "created_at", "deleted_at", etc.

That's possible, but it will remove the option of using code gen for larger changes. We need these sorted alphabetically so that they match the list of fields in the other methods.